CONTRIBUTING.md

```diff
@@ -107,6 +107,9 @@ There are three places to get help:
 - Badges should not obtain data by scraping web pages - these are likely to break frequently.
   Whereas API publishers are incentivised to maintain a stable platform for their users,
   authors of web pages have no such incentive.
+- Badges may require users to specify a token in the badge URL as long it is scoped only to
+  fetching status and doesn't expose any sensitive information. Generating a token with the
```

Comment:
May be worth more general verbiage here to account for other types of data points
```suggestion
  fetching information and doesn't expose any sensitive information. Generating a token with the
```

Reply:
Yup, makes sense!